### DISCUSSION TITLE
Support for 3D graphics?

### DISCUSSION THREAD

**Discussion by kucint** [dotnet/System.Graphics#252] [Q&A]
I suppose Microsoft.Maui.Graphics encapsulates 2D graphics only. What about 3D graphics?
Do you have any plans to support it as well?
if yes, could you deliver any information on this topic, please?
if no, How am I suppose to integrate 3d graphics into Maui App?

FYI: I develop a cross platform Desktop app, Mobile support is out of my scope of interest.

**Comment by akhanalcs** [comment]
Avalonia supports 3D graphics. Check them out.

**Reply by kucint** [reply]
@affableashish, thanks for info, yes I take in account Avalonia as well, but currently the company I am working for wants to invest primary in MAUI. Any news about 3D graphics in MAUI?
